aws-env.dev

```diff
@@ -1,13 +1,13 @@
-export BUGZILLA_ADMIN_KEY=$(aws --profile uneet-dev ssm get-parameters --names BUGZILLA_ADMIN_KEY --with-decryption --query Parameters[0].Value --output text)
-export BUGZILLA_URL=https://dashboard.dev.unee-t.com
-export MONGO_PASSWORD=$(aws --profile uneet-dev ssm get-parameters --names MONGO_PASSWORD --with-decryption --query Parameters[0].Value --output text)
+export BUGZILLA_ADMIN_KEY=$(aws --profile uneet-dev ssm get-parameters --names BUGZILLA_ADMIN_KEY_INS --with-decryption --query Parameters[0].Value --output text)
+export BUGZILLA_URL=https://dashboard-ins.dev.unee-t.com
+export MONGO_PASSWORD=$(aws --profile uneet-dev ssm get-parameters --names MONGO_PASSWORD_INS --with-decryption --query Parameters[0].Value --output text)
 export MONGO_CONNECT="dev-shard-00-00-qxxao.mongodb.net:27017,dev-shard-00-01-qxxao.mongodb.net:27017,dev-shard-00-02-qxxao.mongodb.net:27017/test?ssl=true&replicaSet=Dev-shard-0&authSource=admin"
 export MYSQL_DATABASE=bugzilla
-export MYSQL_HOST=auroradb.dev.unee-t.com
-export MYSQL_PASSWORD=$(aws --profile uneet-dev ssm get-parameters --names MYSQL_PASSWORD --with-decryption --query Parameters[0].Value --output text)
+export MYSQL_HOST=auroradb-ins.dev.unee-t.com
+export MYSQL_PASSWORD=$(aws --profile uneet-dev ssm get-parameters --names MYSQL_PASSWORD_INS --with-decryption --query Parameters[0].Value --output text)
 export MYSQL_PORT=3306
 export MYSQL_USER=bugzilla
-export PARAMS_URL=https://raw.githubusercontent.com/unee-t/bugzilla-customisation/master/params/dev-params.json
-export SES_SMTP_PASSWORD=$(aws --profile uneet-dev ssm get-parameters --names SES_SMTP_PASSWORD --with-decryption --query Parameters[0].Value --output text)
-export SES_SMTP_USERNAME=$(aws --profile uneet-dev ssm get-parameters --names SES_SMTP_USERNAME --query Parameters[0].Value --output text)
+export PARAMS_URL=https://raw.githubusercontent.com/Unee-T-INS/bugzilla-customisation/master/params/dev-params.json
+export SES_SMTP_PASSWORD=$(aws --profile uneet-dev ssm get-parameters --names SES_SMTP_PASSWORD_INS --with-decryption --query Parameters[0].Value --output text)
+export SES_SMTP_USERNAME=$(aws --profile uneet-dev ssm get-parameters --names SES_SMTP_USERNAME_INS --query Parameters[0].Value --output text)
 export SES_VERIFIED_SENDER=dev.case@unee-t.com
```

Comment:
if you've created a dedicated SES service for Unee-T INS (which is good) I think you also have to create a new SES_VERIFIED_SENDED too.
maybe you can configure `dev.case.ins@unee-t.com` as an authorized sender in SES?

Reply:
I can config dev.case.ins@unee-t.com. But we have to create that email and verify it first.

Reply:
I have created the following email addresses as forwarders to the email address osam@unee-t.com:
- dev.ins@unee-t.com: generic email address for the DEV environment of the Unee-T INS installation
- dev.case.ins@unee-t.com: email address for the MEFE side of DEV environment of the Unee-T INS installation
- dev.dashboard.ins@unee-t.com: email address for the BZFE side of DEV environment of the Unee-T INS installation